app/src/main/cpp/src/components/transport_manager/src/bluetooth_le/ble_server.cc

```diff
@@ -0,0 +1,92 @@
+#include "transport_manager/bluetooth_le/ble_server.h"
+#include <thread>
+#include "utils/logger.h"
+
+SDL_CREATE_LOG_VARIABLE("Ble_Server")
+
+namespace{
+    const char* socket_name =  "./localBleWriter";
+
+    char buffer[256];
+    bool thread_stop = false;
```

Comment:
@ychernysheva what is the purpose of that variable?

Reply:
For now I replace it with true to make this cycle infinite